removing handlebar

marcellkiss opened the pull request:
First of all, thanks for the awesome tutorial, the code is really elegant, I really liked it!
I also observed, that even if handlebar is included it's not in use. As this is a tutorial, I think it's better to remove the unused parts.

jameshibbard commented:
Thanks for the kind words :)
But isn't this just changing the templating engine for the sake of it? Or did I miss something?

marcellkiss commented:
I changed to plain html, and dropped the handlebar dependencies from package.json. I think this makes your example just more simple and pure - as handlebar wasn't a necessary dependency in the example.

jameshibbard commented:
The idea is that it's supposed to be extensible, starting with generating a CSRF token (left out for obvious reasons!). It makes it far easier to build upon it if there's already a templating engine installed.
